client/scripts/app.ts

```diff
@@ -157,7 +158,7 @@ export function handleUpdateEmailConfirmation() {
   }
 }
 
-export async function selectCommunity(c?: CommunityInfo, node?: NodeInfo): Promise<boolean> {
+export async function selectCommunity(c?: CommunityInfo): Promise<boolean> {
```

Comment:
Why did you need to do this?

Reply:
i previously used node as argument in selectCommunity function because cw-protocol was offchain community. To change network for CWP, i needed to select Node in selectCommunity funcation. but since i updated it to Chain, we don't need to change Node in selectCommunity.